demo/index.html

```diff
@@ -7,16 +7,16 @@
     <link
       rel="stylesheet"
       type="text/css"
-      href="../node_modules/prismjs/themes/prism.css"
+      href="https://unpkg.com/prismjs@latest/themes/prism.css"
```

Comment:
Looks like prism js is getting updated about every 3 months. What do you think about locking this down to a known working version?

Reply:
Looking through the commits, I don't see this being a huge issue. Also, this is the demo page in the repo, so very low risk if there are any changes.